kinto/core/storage/memory.py

```diff
@@ -67,6 +67,36 @@ def extract_record_set(self, records,
                                   pagination_rules=pagination_rules,
                                   limit=limit)
 
+    def _manage_collection_timestamp(self, collection_timestamp,
+                                     record, modified_field, last_modified):
```

Comment:
If you expose it, it should not start with `_`, and probably have a doctstring with expected parameters and returned value ;)

Reply:
So we would have `bump_timestamp` and `_bump_timestamp`?